### ISSUE TITLE
Conflict with master node configuration in Wazuh cluster using Kubernetes

### ISSUE THREAD

**Issue by SitoRBJ** [wazuh/wazuh-docker#83]
Hello team,

When deploying the Wazuh cluster using Kubernetes we have a problem in the master node, which does not know how to solve the pod "hostname" in the cluster configuration in `ossec.conf`. 

ConfigMap: https://github.com/wazuh/wazuh-kubernetes/blob/master/wazuh_managers/wazuh-master-conf.yaml#L325

```
        <nodes>
            <node>wazuh-manager-master-0.wazuh-cluster.wazuh.svc.cluster.local</node>
        </nodes>
```

Error:

`ERROR: Error 3004 - Error in cluster configuration: Master IP not valid. Valid ones are: 11.22.33.44`

Cause: https://github.com/wazuh/wazuh/blob/v3.7.2/framework/wazuh/cluster/cluster.py#L87-L88

We are going to implement an alternative solution in Docker. 

Regards,

Alfonso Ruiz-Bravo

**Comment by jesuslinares** [comment]
Thanks @SitoRBJ.

We will keep this issue open until it is fixed in Wazuh.

**Comment by SitoRBJ** [comment]
Hello @jesuslinares,

Apparently this problem was solved in the PR [#2227](https://github.com/wazuh/wazuh/pull/2227)

We have performed the PR #99 to remove the alternative solution that is already invalidated. 

Kind regards,

Alfonso Ruiz-Bravo
